sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1479,6 +1479,7 @@ public partial class SearchIndexer
         public SearchIndexer(string name, string dataSourceName, string targetIndexName) { }
         public string DataSourceName { get { throw null; } set { } }
         public string Description { get { throw null; } set { } }
+        public Azure.Search.Documents.Indexes.Models.SearchResourceEncryptionKey EncryptionKey { get { throw null; } set { } }
```

Comment:
For Storage, we made Customer Provided Keys one of the ClientOptions so you didn't have to thread it around every API.  It looks like this is only used selectively for resource creation in Search so I'm okay with the difference but wanted to call it out.

Reply:
If it was for the client-scope I'd be inclined to agree, but the same client could be used for multiple encrypted and non-encrypted (doubly, that is) indexes and related resources. Maybe that's not practical, but at the time we GA'd `EncryptionKey` as a property on indexes and synonym maps was already established as a pattern, so we stuck with that here.